yal/contentrepo.py

```diff
@@ -171,7 +181,7 @@ async def get_page_details(user, page_id, message_id, suggested=False):
             except HTTP_EXCEPTIONS as e:
                 if i == 2:
                     logger.exception(e)
-                    return True, []
+                    return True, {}
```

Comment:
There wasn't any typing on these functions, so I added some, which revealed some issues that I also fixed